app/src/main/java/com/keylesspalace/tusky/TabPreferenceActivity.kt

```diff
@@ -421,6 +414,5 @@ class TabPreferenceActivity : BaseActivity(), Injectable, ItemInteractionListene
 
     companion object {
         private const val MIN_TAB_COUNT = 2
```

Comment:
Better have 1 here?

(2 seems arbitrary, 1 is a "guard against total misconfiguration")